internal/command/user_v2_human.go

```diff
@@ -415,6 +414,14 @@ func (c *Commands) changeUserPhone(ctx context.Context, cmds []eventstore.Comman
 	ctx, span := tracing.NewSpan(ctx)
 	defer func() { span.End() }()
 
+	if phone.Remove {
+		if phone.Number != "" && phone.Number != wm.Phone {
```

Comment:
IMO this will result in confusion, I would remove this check here.
There is a `Validate` function on the `ChangeHuman` specifically for this situation, we don't need to process the request if we already know at the beginning that the arguments are invalid.
You can also add a `Validate` function to the `Phone` struct, if you want to check it with the struct content directly.

Reply:
Makes sense, adjusted 👍🏻